src/MADE.UI.Controls.DropDownList/MADE.UI.Controls.DropDownList.csproj

```diff
@@ -10,7 +10,7 @@
     </Description>
     <PackageTags>MADE UI Views Controls UWP DropDown List Multiselect</PackageTags>
     <GenerateLibraryLayout>true</GenerateLibraryLayout>
-    <RootNamespace>MADE.UI.Controls</RootNamespace>
+    <RootNamespace>MADE.UI.Controls.DropDownListControl</RootNamespace>
```

Comment:
Why has the root namespace here changed?

Reply:
@tom-made so these changes were made due to the way that generated files are made in Uno. 

Because we made the decision to have each individual control as its own lib to allow devs to pick and choose components, there is a conflict with each project having the same root namespace. 

This is a workaround for that but we will need to keep an eye on additional controls to make sure all the code within the projects sit at the `MADE.UI.Controls` namespace.

Reply:
No worries, just wanted to make sure as I couldn't see why 👍